### ISSUE TITLE
401 error code when I try to get my Autodesk Profile

### ISSUE THREAD

**Issue by efdiloreto** [autodesk-platform-services/aps-hubs-browser-dotnet#2]
I'm getting this error when I'm try to get my profile:

`{"type":"https://tools.ietf.org/html/rfc7235#section-3.1","title":"Unauthorized","status":401,"traceId":"00-000a46ddfb59e6fae081ba14aeb9f341-9e121c5e0f22a36b-00"}`

I think the error is coming from here because it's not giving any tokens:

https://github.com/autodesk-platform-services/aps-hubs-browser-dotnet/blob/dd6e0a0c2059824e1c2795748367234d63367f04/Controllers/AuthController.cs#L75

I have to say that this was working yesterday for me. But right now it's not working anymore.

Any suggestions?

**Comment by petrbroz** [comment]
Can you still reproduce the issue today? If so, have you made any other changes to the code, or to your APS application (for example, generating a new Client Secret)?

**Comment by efdiloreto** [comment]
Yes, I still have the issue. I've tried with other APS account and the same is happening. I've also deleted cookies, etc.

I'm seeing I have negative tokens in my APS account. Does executing a 3 legged auth consume tokens?

**Comment by petrbroz** [comment]
The auth API does not consume any tokens but it's possible that a negative balance could be affecting what your app can or cannot do. To confirm or rule out this possibility, could you try and test your app with a different APS account (even a trial one)?
